htdocs/js/main.js

```diff
@@ -61,8 +61,9 @@ $(document).ready(function() {
 
     $('#shortcut_form').submit(function(e) {
         var target = $('#shortcut');
-        if (!Validation.isNumberOnly(target.val())) {
-            alert('Please enter numbers only');
+        target.val(target.val().replace(/\D/g,''));
+        if (Validation.isWhitespace(target.val())) {
```

Comment:
but `target` will be empty, no need for `Validation.isWhitespace()` call

also, for DRY, create just `var value = target.val().replace(/\D/g,'')` instead of calling `target.val()` repeatedly?